tripy/tests/backend/mlir/test_compiler.py

```diff
@@ -53,9 +53,9 @@ def test_reason_context(self):
         with FlatIRTensor.context(["This is the first level of context"]):
             with FlatIRTensor.context(["This is the second level of context"]):
                 # We need to emit an error from one of the internally created `FlatIRTensor`s to see the context
-                a = tp.ones(1)
-                b = tp.ones(1)
-                trace = Trace([a + b])
+                a = tp.ones((1,))
+                b = tp.ones((1,))
+                trace = Trace([a[0] + b[0]])
```

Comment:
I think the intention was to add the entire tensor? Not sure if it actually matters either way for this test.

Reply:
The test works either way. I'll keep it as it was.